exporter/util.py

```diff
@@ -93,14 +93,14 @@ def default_files(cls):
             "xlsx": {"full": False, "years": set()},
         }
 
-    def __init__(self, *components, export_type: str = "json"):
+    def __init__(self, *components, export_type: str = "json", lockfile_suffix=""):
```

Comment:
Let's annotate, since `export_type` is annotated.

```suggestion
    def __init__(self, *components, export_type: str = "json", lockfile_suffix: str = ""):
```